ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiVersionService.scala

```diff
@@ -64,7 +65,9 @@ private[apiserver] final class ApiVersionService private (
       experimental = Some(
         ExperimentalFeatures.of(
           selfServiceErrorCodes =
-            Option.when(enableSelfServiceErrorCodes)(ExperimentalSelfServiceErrorCodes()),
+            Option.when(enableSelfServiceErrorCodes)(ExperimentalSelfServiceErrorCodes()): @nowarn(
```

Comment:
As hinted above, we can `reserve` this field completely and drop it from the protocol